Lesson2/HomeWork/Task2/Programm.java

```diff
@@ -1,7 +1,9 @@
 class Programm{
 	public static void main(String[] args){
-		Report report = new Report();
+		Report report = new Report(new DisplayReport);
 		report.calculate();
 		report.output();
+		report.setSendReport(new PrintReport());

```

Comment:
Лучше сделать наоборот, передать report в PrintReport. Так код будет проще.
 ```suggestion
		report.setSendReport(new PrintReport());
		SendOutput reportOutput = new PrintReport();
		reportOutput.output(report)		
```